app/modules/asset_groups/metadata.py

```diff
@@ -137,6 +137,22 @@ def _validate_lat_long(cls, dictionary, error_str):
                 log,
                 f'Need both or neither of decimalLatitude and decimalLongitude in {error_str}',
             )
+
+        # Both may be null but if only one is, it's a failure
+        if (
+            'null' == dictionary['decimalLatitude']
+            and 'null' == dictionary['decimalLongitude']
+        ):
+            return
+        if (
+            'null' == dictionary['decimalLatitude']
+            or 'null' == dictionary['decimalLongitude']
+        ):
+            raise AssetGroupMetadataError(
```

Comment:
pretty sure these `'null'` comparisons need to instead be comparing against `None`, as that is what the json `{"decimalLatitude": null, ...}` gets deserialized to.   thus, more like `if not dictionary['decimalLatitude']...`

Reply:
> pretty sure these `'null'` comparisons need to instead be comparing against `None`, as that is what the json `{"decimalLatitude": null, ...}` gets deserialized to. thus, more like `if not dictionary['decimalLatitude']...`

exactly what I just found and why this is draft again